docker-compose-setup/docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+services:
+  coatrack-postgres:
+    image: postgres:9.4.26
+    container_name: coatrack-postgres
+    volumes:
+      - coatrack-postgres:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: password1234
+  coatrack-config-server:
+    image: coatrack/coatrack-config-server:2.0.0-SNAPSHOT
+    container_name: coatrack-config-server
+    depends_on:
+      - coatrack-postgres
+    ports:
+      - 8998:8998
+    entrypoint: ["sh", "-c", "java -Djava.security.egd=file:/dev/./urandom \
+      -jar /opt/coatrack/lib/coatrack-config-server.jar \
+      --spring.datasource.url=jdbc:postgresql://coatrack-postgres/ygg_config_server \
+      --spring.datasource.password=password1234"]
+  coatrack-admin:
+    image: coatrack/coatrack-admin:2.0.0-SNAPSHOT
+    container_name: coatrack-admin
+    volumes:
+      - coatrack-admin:/home
+    depends_on:
+      - coatrack-config-server
+      - coatrack-postgres
+    ports:
+      - 8080:8080
+    entrypoint: [ "sh", "-c", "sleep 5 && java -Djava.security.egd=file:/dev/./urandom \
+      -jar /opt/coatrack/lib/coatrack-admin.jar \
+      --ygg.admin.gitService.url=https://bitbucket.org/perezdf/anthill-settings.git \
```

Comment:
Before merging this to coatrack master, we should either put placeholders here, to be filled by users, or we should include a git repository in the docker-compose, to remove the external dependency to bitbucket.

Reply:
I totally agree on that. I think we should discuss the details on the next occasion.